patches/master_patch.patch

```diff
@@ -1598,77 +1649,60 @@ index 5ccfa4398c7180ff9268d6a909668a2d54f8a89c..35f0bff7bd52db74cb421abd4edbec60
        storage_partition_impl->site_for_service_worker().is_empty()) {
      storage_partition_impl->set_site_for_service_worker(
          site_instance->GetSiteURL());
-diff --git a/content/browser/renderer_host/render_widget_host_ns_view_bridge.h b/content/browser/renderer_host/render_widget_host_ns_view_bridge.h
-index 3761be5f730dd73662acdd016ee098d64556b09c..e3066953627059ec96ebd6b159db7251c4bd38cf 100644
---- a/content/browser/renderer_host/render_widget_host_ns_view_bridge.h
-+++ b/content/browser/renderer_host/render_widget_host_ns_view_bridge.h
-@@ -31,6 +31,7 @@ enum class DomCode;
- 
- namespace content {
- 
-+class RenderWidgetHostView;
- class RenderWidgetHostNSViewClient;
- class WebCursor;
- 
-@@ -117,6 +118,10 @@ class RenderWidgetHostNSViewBridge {
-   // Stop intercepting keyboard events.
-   virtual void UnlockKeyboard() = 0;
- 
-+#if !defined(MUON_CHROMIUM_BUILD)
```

Comment:
I'm not sure I understand what's going on with this change

Reply:
I've taken care of it in https://github.com/brave/muon/pull/642/commits/18115e77b3a09b4800e18c7f5fe4adda44e920c4